package.json

```diff
@@ -14,14 +14,15 @@
     "lint-staged": "lerna run lint-staged --stream --",
     "lint": "lerna run lint --stream",
     "setup": "yarn && yarn bootstrap && yarn build",
-    "setup:example": "yarn --cwd example",
     "specify:pull": "specify pull",
     "specify:sync": "specify sync",
     "storybook": "lerna run storybook --stream",
-    "test:example": "yarn --cwd example test --watchAll=false",
-    "test:e2e:ci": "lerna run test:e2e:ci",
     "test:e2e": "lerna run test:e2e --stream",
+    "test:e2e:ci": "lerna run test:e2e:ci",
+    "test:e2e:watch": "yarn --cwd packages/strapi-design-system test:e2e:watch",
     "test": "lerna run test --stream",
+    "test:watch": "yarn --cwd packages/strapi-design-system test:watch",
```

Comment:
We should use lerna with the `--scope` flag instead.

Here's the [documentation](https://www.npmjs.com/package/@lerna/filter-options)